app/app.model.js

```diff
@@ -49,9 +49,27 @@ function selectAllArticles() {
       return article.rows;
     });
 }
+
+function selectAllComments(article_id) {
+  return db
+    .query(
+      ` SELECT * FROM comments WHERE article_id = $1 ORDER BY comments.created_at DESC`,
+      [article_id]
+    )
+    .then((comment) => {
+      if (comment.rows.length === 0) {
```

Comment:
Is it an error if an article has no comments? What about a brand new article that I've only just written, would that have any comments?